{"uid":"8dedb09243f374fb","name":"test_accept_order_without_order_id","fullName":"tests.test_return_order.TestCreatingCourier#test_accept_order_without_order_id","historyId":"d6ffe158eb016c9abb6993e0eb78808f","time":{"start":1760439189777,"stop":1760439190143,"duration":366},"status":"failed","statusMessage":"AssertionError: Ожидался статус 400, но получили 404\nassert 404 == 400\n +  where 404 = <Response [404]>.status_code","statusTrace":"self = <test_return_order.TestCreatingCourier object at 0x000001D650E11940>, prepared_courier = 629405\n\n    def test_accept_order_without_order_id(self, prepared_courier):\n        with allure.step(\"Принять заказ не передавая id заказа\"):\n            resp = requests.put(f'{Url.url}{Url.put_order}/?courierId={prepared_courier}')  # Без order_id\n    \n>           assert resp.status_code == 400, f\"Ожидался статус 400, но получили {resp.status_code}\"\nE           AssertionError: Ожидался статус 400, но получили 404\nE           assert 404 == 400\nE            +  where 404 = <Response [404]>.status_code\n\ntests\\test_return_order.py:38: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1760438909140,"stop":1760438909179,"duration":39},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"attachmentStep":false,"attachmentsCount":0,"stepsCount":0},{"name":"prepared_courier","time":{"start":1760439189022,"stop":1760439189776,"duration":754},"status":"passed","steps":[{"name":"Создаем тестового курьера","time":{"start":1760439189022,"stop":1760439189775,"duration":753},"status":"passed","steps":[{"name":"Создаем курьера","time":{"start":1760439189022,"stop":1760439189379,"duration":357},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"attachmentStep":false,"attachmentsCount":0,"stepsCount":0},{"name":"Получаем ID курьера через авторизацию","time":{"start":1760439189380,"stop":1760439189773,"duration":393},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"attachmentStep":false,"attachmentsCount":0,"stepsCount":0}],"attachments":[],"parameters":[],"hasContent":true,"shouldDisplayMessage":false,"attachmentStep":false,"attachmentsCount":0,"stepsCount":2}],"attachments":[],"parameters":[],"hasContent":true,"shouldDisplayMessage":false,"attachmentStep":false,"attachmentsCount":0,"stepsCount":3},{"name":"create_cour","time":{"start":1760439189022,"stop":1760439189022,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"attachmentStep":false,"attachmentsCount":0,"stepsCount":0},{"name":"post_cour","time":{"start":1760439189022,"stop":1760439189022,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"attachmentStep":false,"attachmentsCount":0,"stepsCount":0}],"testStage":{"status":"failed","statusMessage":"AssertionError: Ожидался статус 400, но получили 404\nassert 404 == 400\n +  where 404 = <Response [404]>.status_code","statusTrace":"self = <test_return_order.TestCreatingCourier object at 0x000001D650E11940>, prepared_courier = 629405\n\n    def test_accept_order_without_order_id(self, prepared_courier):\n        with allure.step(\"Принять заказ не передавая id заказа\"):\n            resp = requests.put(f'{Url.url}{Url.put_order}/?courierId={prepared_courier}')  # Без order_id\n    \n>           assert resp.status_code == 400, f\"Ожидался статус 400, но получили {resp.status_code}\"\nE           AssertionError: Ожидался статус 400, но получили 404\nE           assert 404 == 400\nE            +  where 404 = <Response [404]>.status_code\n\ntests\\test_return_order.py:38: AssertionError","steps":[{"name":"Принять заказ не передавая id заказа","time":{"start":1760439189777,"stop":1760439190142,"duration":365},"status":"failed","statusMessage":"AssertionError: Ожидался статус 400, но получили 404\nassert 404 == 400\n +  where 404 = <Response [404]>.status_code\n","statusTrace":"  File \"C:\\Users\\ilua2\\project\\qa_Sprint_7\\tests\\test_return_order.py\", line 38, in test_accept_order_without_order_id\n    assert resp.status_code == 400, f\"Ожидался статус 400, но получили {resp.status_code}\"\n","steps":[],"attachments":[],"parameters":[],"hasContent":true,"shouldDisplayMessage":true,"attachmentStep":false,"attachmentsCount":0,"stepsCount":0}],"attachments":[],"parameters":[],"hasContent":true,"shouldDisplayMessage":true,"attachmentStep":false,"attachmentsCount":0,"stepsCount":1},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_return_order"},{"name":"subSuite","value":"TestCreatingCourier"},{"name":"host","value":"Ilya_Volkov"},{"name":"thread","value":"5776-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_return_order"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[]}],"tags":[]},"source":"8dedb09243f374fb.json","parameterValues":[]}